apps/appsets/argocd-operators-project.yaml

```diff
@@ -20,6 +20,11 @@ spec:
     server: '*'
   - namespace: 'rook-ceph'
     server: '*'
+  - namespace: 'monitoring'
+    server: '*'
+  # kube-system is used by kube-prometheus-stack
```

Comment:
What's being installed into that namespace?

Reply:
Some default exporters and such: https://github.com/search?q=repo%3Aprometheus-community%2Fhelm-charts+path%3A%2F%5Echarts%5C%2Fkube-prometheus-stack%5C%2F%2F+kube-system&type=code